hknweb/tours/migrations/0001_initial.py

```diff
@@ -0,0 +1,35 @@
+# Generated by Django 2.2.8 on 2020-03-25 05:37
+
+import datetime
+import django.core.validators
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='DepTour',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('name', models.CharField(default='', max_length=255)),
+                ('email', models.EmailField(default='', max_length=255)),
+                ('confirm_email', models.EmailField(default='', max_length=255)),
+                ('desired_time', models.TimeField(verbose_name='Desired Time')),
+                ('date', models.DateField(default=datetime.date.today, verbose_name='Desired Date')),
+                ('phone', models.CharField(default='', max_length=15, validators=[django.core.validators.RegexValidator(message="Enter a valid 10-digit phone number in the format:'+999999999'", regex='^\\+?1?\\d{10,15}$')])),
```

Comment:
The example (`+999999999`) only looks like it has 9 digits? Ultimately I'd do something like `+1234567890` as an example so it's clearer how many digits there are at a glance, but that's not a big deal